gapis/api/gles/api/image_format.api

```diff
@@ -470,23 +470,23 @@ sub GLenum GetSizedFormatFromTuple(GLenum unsizedFormat, GLenum type) {
     case GL_LUMINANCE_ALPHA: switch type {
       case GL_UNSIGNED_BYTE: GL_LUMINANCE8_ALPHA8_EXT
       @if(Extension.GL_OES_texture_half_float)
-      case GL_HALF_FLOAT_OES: GL_LUMINANCE_ALPHA16F_EXT
+      case GL_HALF_FLOAT, GL_HALF_FLOAT_OES: GL_LUMINANCE_ALPHA16F_EXT
       @if(Extension.GL_OES_texture_float)
       case GL_FLOAT: GL_LUMINANCE_ALPHA32F_EXT
       default: GL_NONE
     }
     case GL_LUMINANCE: switch type {
       case GL_UNSIGNED_BYTE: GL_LUMINANCE8_EXT
       @if(Extension.GL_OES_texture_half_float)
-      case GL_HALF_FLOAT_OES: GL_LUMINANCE16F_EXT
+      case GL_HALF_FLOAT, GL_HALF_FLOAT_OES: GL_LUMINANCE16F_EXT
       @if(Extension.GL_OES_texture_float)
       case GL_FLOAT: GL_LUMINANCE32F_EXT
       default: GL_NONE
     }
     case GL_ALPHA: switch type {
       case GL_UNSIGNED_BYTE: GL_ALPHA8_EXT
       @if(Extension.GL_OES_texture_half_float)
-      case GL_HALF_FLOAT_OES: GL_ALPHA16F_EXT
+      case GL_HALF_FLOAT, GL_HALF_FLOAT_OES: GL_ALPHA16F_EXT
```

Comment:
Doesn't CheckTextureDataType (line 503) need update as well?

Reply:
No. that one already handles both constants. Make sure to read through the whole function to notice.